### ISSUE TITLE
GHA Matrix values in Slack Message

### ISSUE THREAD

**Issue by smcvb** [voxmedia/github-action-slack-notify-build#39]
First off, thanks for building this GitHub Action.
I've quite recently started using it, but it does exactly what it promises.

Now, my projects "Build and Test" GHA uses the `strategy.matrix` to define two main builds, one for JDK8 and another for JDK11.
However, from the perspective of the Slack messages being published, I simply see two builds for seemingly exactly the same thing. Obviously, this isn't the case, as the former would be JDK8 and the latter JDK11.

Up till now, I haven't found a way to configure this for this GHA.
Am I missing something obvious here, or is this a potential feature on the roadmap?

**Comment by geoffreywiseman** [comment]
Yes, I'd like this as well. Although I guess I would slightly prefer it to be a single message that gets updated with results from both matrix builds, although that does sound more complicated, and might be harder to achieve.
